[Docs] Clarify that explore underlying data is not enabled (#105693)
elastic/kibana: docs/user/dashboard/dashboard.asciidoc

[[explore-the-underlying-documents]]
== Explore the underlying documents

> A {kib} administrator can add additional interactions for {kib} users to take from their dashboard panels.

Reviewer: ```suggestion
You can add additional interactions that allow you to open *Discover* from dashboard panels. To use the interactions, the panel must use only one index pattern.
```

> These additional interactions allow users to open *Discover* from the dashboard.

Reviewer: ```suggestion
```

> Panel context:: Adds an *Explore underlying data* action to each panel's context menu. This action does not carry the filters from your dashboard to *Discover*.

Reviewer: ```suggestion
Panel interaction:: Opens the panel data in *Discover*. The panel interaction does not open the filters from your dashboard to *Discover*.
```

+

> Enable by setting <<settings-explore-data-in-context,`xpack.discoverEnhanced.actions.exploreDataInContextMenu.enabled`>>.

Reviewer: ```suggestion
To enable panel interactions, refer to <<settings-explore-data-in-context,`xpack.discoverEnhanced.actions.exploreDataInContextMenu.enabled`>>.
```

> Series context::

Reviewer: ```suggestion
Series interaction::
```

> Adds an action to each series in a panel. Instead of filtering automatically, users choose to filter or open *Discover*.

Reviewer: ```suggestion
Opens the series data in *Discover*. 
```

+

> Enable by setting <<settings-explore-data-in-chart,`xpack.discoverEnhanced.actions.exploreDataInChart.enabled`>>.

Reviewer: ```suggestion
To enable series interactions, refer to <<settings-explore-data-in-chart,`xpack.discoverEnhanced.actions.exploreDataInChart.enabled`>>.
```

> Both actions require that the panel uses only one index pattern.

Reviewer: ```suggestion
```

> NOTE: Prior to {kib} 7.13, the panel context menu was enabled by default.

Reviewer: ```suggestion
NOTE: In {kib} 7.13 and earlier, the panel interaction menu was enabled by default.
```
